lib/jsdom/browser/Window.js

```diff
@@ -148,6 +156,10 @@ function setupWindow(windowInstance, { runScripts }) {
 
   windowConstructor.prototype = windowPrototype;
   Object.setPrototypeOf(windowInstance, windowPrototype);
+  if (runScripts === "outside-only" || runScripts === "dangerously") {
+    const global = vm.runInContext("this", windowInstance);
+    Object.setPrototypeOf(global, windowPrototype);
```

Comment:
Note to self: we could try also setting `EventTarget.prototype.__proto__` to the `Object.prototype` from the vm context. Might be best to do with a major version bump...

Reply:
Yes, I assume this could be a big change so I'd leave it from this PR...